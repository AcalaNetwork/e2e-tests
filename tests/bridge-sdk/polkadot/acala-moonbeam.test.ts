import { bridgeSDKTest } from '../shared'

bridgeSDKTest({
  from: 'acala',
  to: 'moonbeam',
  token: 'AUSD',
})
