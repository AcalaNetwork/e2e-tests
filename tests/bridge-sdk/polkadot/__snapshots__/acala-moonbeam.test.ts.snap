// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`'acala' to 'moonbeam' using bridgeSDK cross-chain 'AUSD' > after 1`] = `
{
  "address": "0x4E7440dB498561A46AAa82b9Bc7d2D5162b5c27B",
  "decimals": 18,
  "fromChain": 8,
  "toChain": "(rounded 1.98)",
}
`;

exports[`'acala' to 'moonbeam' using bridgeSDK cross-chain 'AUSD' > fee 1`] = `"(rounded 0.02)"`;

exports[`'acala' to 'moonbeam' using bridgeSDK cross-chain 'AUSD' > initial 1`] = `
{
  "address": "0x4E7440dB498561A46AAa82b9Bc7d2D5162b5c27B",
  "decimals": 18,
  "fromChain": 10,
  "toChain": 0,
}
`;
